web/packages/teleport/src/Console/DocumentSsh/DocumentSsh.tsx

```diff
@@ -77,29 +78,39 @@ export default function DocumentSsh({ doc, visible }: PropTypes) {
           afterClose={handleCloseFileTransfer}
           backgroundColor={colors.primary.light}
           transferHandlers={{
-            getDownloader: async (location, abortController) =>
-              getHttpFileTransferHandlers().download(
+            getDownloader: async (location, abortController) => {
+              const wanResponse = await auth.getAssertionResponseIfRequired({
```

Comment:
for here and below how are we handling possible errors?

Reply:
the errors are handled within the `getBaseXHRRequest`  [onerror](https://github.com/gravitational/teleport/blob/5df4323f9483e0514a8a0c7eb848d3dc4c471759/web/packages/teleport/src/Console/DocumentSsh/httpFileTransferHandlers.ts#L127) and [getFileReaderErrorAsText](https://github.com/gravitational/teleport/blob/5df4323f9483e0514a8a0c7eb848d3dc4c471759/web/packages/teleport/src/Console/DocumentSsh/httpFileTransferHandlers.ts#L148)

Reply:
So the errors will be displayed like any other errors that can happen during a file transfer, do I understand this correctly?

Reply:
This flow is removed now so it isn't relevant. I've moved the check into an export of `getHttpFileTransferHandlers` and reuse the abort controller. Errors will now display like any other, yes